doc/source/user/absolute_beginners.rst

```diff
@@ -0,0 +1,2453 @@
+
+  1. If `buffer` is None, then only `shape`, `dtype`, and `order`
+     are used.
+  2. If `buffer` is an object exposing the buffer interface, then
+     all keywords are interpreted.
+
+  No ``__init__`` method is needed because the array is fully initialized
+  after the ``__new__`` method.
+
+  Examples
+  --------
+  These examples illustrate the low-level `ndarray` constructor.  Refer
+  to the `See Also` section above for easier ways of constructing an
+  ndarray.
+
+  First mode, `buffer` is None:
+
+  >>> np.ndarray(shape=(2,2), dtype=float, order='F')
+  array([[ -1.13698227e+002,   4.25087011e-303],
+         [  2.88528414e-306,   3.27025015e-309]])         #random
+
+  Second mode:
+
+  >>> np.ndarray((2,), buffer=np.array([1,2,3]),
+  ...            offset=np.int_().itemsize,
+  ...            dtype=int) # offset = 1*itemsize, i.e. skip first element
+  array([2, 3])
+
+This also works for functions and other objects that **you** create. Just remember to include a docstring with your function using a string literal (**""" """** or **''' '''** around your documentation).
+
+For example, if you create this function:
+
+::
+
+  def double(a):
+    '''Return a * 2'''
+    return a * 2
+
+You can run
+
+::
+
+  double?
+
+Which will return
+
+::
+
+  Signature: double(a)
+  Docstring: Return a * 2
+  File:      ~/Desktop/<ipython-input-23-b5adf20be596>
+  Type:      function
+
+You can reach another level of information by reading the source code of the object you're interested in. Using a double question mark (**??**) allows you to access the source code.
+
+For example, running:
+
+::
+
+  double??
+
+Will return 
+
+::
+
+  Signature: double(a)
+  Source:   
+  def double(a):
+      '''Return a * 2'''
+      return a * 2
+  File:      ~/Desktop/<ipython-input-23-b5adf20be596>
+  Type:      function
+
+If the object in question is compiled in a language other than Python, using ?? will return the same information as ?. You'll find this with a lot of built-in objects and types, for example:
+
+::
+
+  len?
+
+**Output:**
+
+::
+
+  Signature: len(obj, /)
+  Docstring: Return the number of items in a container.
+  Type:      builtin_function_or_method
+
+and
+
+::
+
+  len??
+
+**Output:**
+
+::
+
+  ​Signature: len(obj, /)
+  Docstring: Return the number of items in a container.
+  Type:      builtin_function_or_method
+
+Have the same output because they were compiled in a programming language other than Python.
+
+
+
+Working with mathematical formulas
+----------------------------------
+
+Implementing mathematical formulas that work on arrays is one of the things that make NumPy so highly regarded in the scientific Python community. 
+
+For example, this is the mean square error formula (a central formula used in supervised machine learning models that deal with regression):
+
+.. image:: images/np_MSE_formula.png
+
+Implementing this formula is simple and straightforward in NumPy:
+
+.. image:: images/np_MSE_implementation.png
```

Comment:
Should this be code rather than an image?

Reply:
I think the visual with the use of matching colors makes this clearer for beginners, but I defer to your judgement

Reply:
You can use color in mathjax as `{ \color{green} \frac{1}{n} }`, or possibly `\color{green}{\frac{1}{n}}`, depending on which version we're using.

Reply:
You can use color in mathjax as `{ \color{green} \frac{1}{n} }`, or possibly `\color{green}{\frac{1}{n}}`, depending on which version we're using.

Reply:
Although I suppose you then end up doing that for all the images in this section, which might be hard work

Reply:
Thanks!

Reply:
I'm inclined to leave this as is, since it's quite a bit of work. We can also change it later if needed - the ~200 kb increase in repo size isn't critical.